File: src/main/java/com/example/projetosisu/Nota.java
package com.example.projetosisu;

public class Nota {
    private int ano;
    private String demanda;   // Ampla concorrência, cotas, etc.
    private double media;
    private int colocacao;

    public Nota(int ano, String demanda, double media, int colocacao) {
        this.ano = ano;
        this.demanda = demanda;
        this.media = media;
        this.colocacao = colocacao;
    }

    public int getAno() { return ano; }
    public String getDemanda() { return demanda; }
    public double getMedia() { return media; }
    public int getColocacao() { return colocacao; }
}

File: src/main/python/sisu_analyser/Nota.py
class Nota:
    def __init__(self, ano: int, demanda: str, media: float, colocacao: int):
        self.__ano = ano
        self.__demanda = demanda
        self.__media = media
        self.__colocacao = colocacao
        
    def get_ano(self):
        return self.__ano
        
    def get_demanda(self):
        return self.__demanda
        
    def get_media(self):
        return self.__media
        
    def get_colocacao(self):
        return self.__colocacao